### PR TITLE
Add support for Ubuntu 64-bit

### PR DESCRIPTION
**Pull request by eloots**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,7 +13,8 @@ collectedLogs.zip
 .bloop/
 .metals/
 .swp
-project/metals.sbt
+metals.sbt
```
Comment (kikiya) [inline .gitignore:16]: has this started building with metals? or supporting various build tools?

Comment (eloots) [reply .gitignore:16]: I'm using VSC with the Metals plugin (alongside IntelliJ) as IDE and this creates these `metals.sbt` files.

File: akka-pi-os-dhcp-32.yml [hunk 2]
```diff
@@ -26,7 +28,9 @@ users:
     sudo: ALL=(ALL) NOPASSWD:ALL
     shell: /bin/bash
     groups: users,docker,video
+    ssh-import-id: None
     plain_text_passwd: akkapi
+    ssh-authorized-keys: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDkrgM1almJcqu6FaKDrscUVghlJOO6m7wayVnmH0bQEv4uQQ9SZmkCegrGJ63iMI8X0xUShkwYLippePpj0RiucL1e9/FtzqSdu6IHaPFJg7w5G6cEV2c1mvyJRHpi86+Yt3t4z67aSw77uAWkNFKShtSEbePfF5WX/LGknxRND6f3db+SmmOSy5w5qgC4GmG31gnJX6j7s0xWRiMicyYdGzKOKB4VkQi2YY7bj7hwri4EUHgIZlYuxST57L68CyTVwnF8c39dK6xflea4fkn+3TlNakFw0rrP3iqy8bkaU2cHnMyWIsYF4fbec1JnKngtIGiungvV9QTJzTpMF7ufEFiYl2uzCZgZGhmXsFDo9WoMjKHTJcbM0cm2OWWOrllOGdY2V580szaJR293jQx0OvNhxA3kTAe1aNmpBOfWhhjxxQsPCaPeHTdNG2L3iN9gYT8mms84ENO7OSUYsiMYMas9EMybyYY3Mvd4NUruIxlRgT84kpgyecC/8iQJwvI7vkdZRLlzQ0B614VCA3RH27ZULODSCPz2oFHFfHrMvdxGa4FTYJCWQFBAltWuPjQ8wmNdnKWX85jlc9MPlJ5WK8VIC1f5hVJd6qjSt009WNEt/y6h9xb5qixgZmBRkGpNe/2fyzx1sn0gyVm90wQXDSoT1fWhxZKJciPuHMvYcw== eric.loots@gmail.com
```
Comment (agolubev) [inline akka-pi-os-dhcp-32.yml:33]: Does this mean we need to import another piece of keys for a local user to be able to log in? Meaning we don't need to generate key with https://github.com/lightbend/Pi-Akka-Cluster/blob/master/docs/Hypriot-OS-Course-Preparation-Instructions.md#configure-password-less-login ?

File: README.md [hunk 3]
```diff
@@ -49,3 +49,14 @@ Details on how to configure such a (TP-Link TL-WR802N) router can be found [here
 - A schematic of the LED strip can be found [here](docs/images/raspberry_led_platine_v2.pdf)
 
 >Instructions for Display assembly are available [here](docs/display/display-assembly-instructions.md)
+
+
+## Running 64-bit Ubuntu
+
+Ubuntu has a 64-bit ARM version with built-in cloud-init support. The `akka-pi-os-dhcp-64.yml` cloud-init file has the required modifications to flash a card for that version of the OS on ARM.
```
Comment (agolubev) [inline README.md:56]: should we mention "ubuntu" in file name?

File: exercise_012_clustered_sudoku_solver/src/main/scala/akkapi/cluster/sudoku/SudokuProblemSender.scala [hunk 4]
```diff
@@ -34,6 +34,8 @@ class SudokuProblemSender private (sudokuSolver: ActorRef[SudokuSolver.Command],
   private val solutionWrapper: ActorRef[SudokuSolver.Response] =
     context.messageAdapter(response => SolutionWrapper(response))
 
+  context.log.info("Dit is een demo ~~>")
```
Comment (agolubev) [inline exercise_012_clustered_sudoku_solver/src/main/scala/akkapi/cluster/sudoku/SudokuProblemSender.scala:37]: Should this be in english?

**Comment by eloots** [comment]
This means that you can add your ssh public key to the file which saves you some time by not having to copy and putting it in the .ssh directory of a freshly installed RPi.

File: akka-pi-os-dhcp-32.yml (hunk 2 above)
Comment (eloots) [reply akka-pi-os-dhcp-32.yml:33]: Yes, this means that you can add your ssh public key to the file which saves you some time by not having to copy and putting it in the .ssh directory of a freshly installed RP!

File: exercise_012_clustered_sudoku_solver/src/main/scala/akkapi/cluster/sudoku/SudokuProblemSender.scala (hunk 4 above)
Comment (eloots) [reply exercise_012_clustered_sudoku_solver/src/main/scala/akkapi/cluster/sudoku/SudokuProblemSender.scala:37]: I removed that line of code (shouldn't have been there anymore when I submitted the PR).

File: README.md (hunk 3 above)
Comment (eloots) [reply README.md:56]: Don't think so: the yaml file is only linked to an os instance that has support for cloud-init.

Comment (eloots) [reply README.md:56]: But you may be right that the way I formulate this links it too tight to Ubuntu. I'll reformulate it a bit.

**Review by agolubev — APPROVED** [review]
Looks good to me

**Pull request merged**